apps/researcher/src/app/[locale]/objects/[id]/metadata.tsx

```diff
@@ -0,0 +1,93 @@
+import {HandThumbUpIcon} from '@heroicons/react/24/outline';
+import {useTranslations} from 'next-intl';
+import {ReactNode} from 'react';
+import {create} from 'zustand';
+
+export const useCurrentOwner = create(() => ({
+  name: '',
+  location: '',
+  dataKey: '',
+}));
+
+interface Props {
+  dataKey: string;
+  children: ReactNode;
+}
+
+export function MetadataContainer({dataKey, children}: Props) {
+  useCurrentOwner.setState({dataKey: dataKey});
+  const t = useTranslations('ObjectDetails');
+
+  return (
+    <div className="flex flex-col gap-4">
+      <div className="flex flex-col xl:flex-row gap-2 xl:gap-10">
+        <div className="w-full xl:w-1/5 border-t pt-4">
+          <div className=" sticky top-0 bg-white py-1">
+            <h3 className="text-lg w-full my-1 flex items-center">
+              {t(dataKey)}
+            </h3>
+            <div className="text-neutral-500 text-sm">
+              {t(`${dataKey}SubTitle`)}
+            </div>
+          </div>
+        </div>
+        <div className="w-full xl:w-4/5 flex flex-col gap-2">{children}</div>
+      </div>
+    </div>
+  );
+}
+
+interface MetadataEntryProps {
+  children: ReactNode;
+  isOwner: boolean;
+  isEmpty: boolean;
```

Comment:
- What does `isEmpty` mean? For example: is there no data?
- Could this component figure out whether there is or isn't data (so that we don't have to use a property for it)?

Reply:
Yes, this means there I no data. I will remove it for now and check if the `children` are null.

Eventually, I want a richer layout for the children (see, for example, the tags on materials in the design). Then `children` will not always be null if there is no data, but I will tackle that problem then. Probably by making a separate `<MetadataTextEntry>` component and `<MetadataThesaurusEntry>`. But I'm still discussing the tag-like layout with Bas. That is why this is not yet integrated.